ui/src/dashboards/components/dashboard_index/DashboardCard.tsx

```diff
@@ -31,31 +31,44 @@ import {resetViews} from 'src/dashboards/actions/views'
 // Utilities
 import {relativeTimestampFormatter} from 'src/shared/utils/relativeTimestampFormatter'
 
-interface PassedProps {
-  dashboard: Dashboard
+interface OwnProps {
+  id: string
```

Comment:
why was this deconstructed?

Reply:
it'll be nicer when labels are a list of IDs.